exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/sdk/trace/exporter.h"
+
+const size_t MAX_BUFFER_SIZE = 100;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace memory
+{
+/**
+ * A in memory exporter that switches a flag once a valid recordable was received
+ * and keeps track of all received spans in memory.
+ */
+class InMemorySpanExporter final : public opentelemetry::sdk::trace::SpanExporter
+{
+public:
+  InMemorySpanExporter(size_t buffer_size = MAX_BUFFER_SIZE)
+      : data_(new opentelemetry::exporter::memory::InMemorySpanData(buffer_size))
+  {}
+
+  std::unique_ptr<sdk::trace::Recordable> MakeRecordable() noexcept override
+  {
+    return std::unique_ptr<sdk::trace::Recordable>(new sdk::trace::SpanData);
```

Comment:
nit: it's more readable to always add the parenthesis when calling constructors/methods.

Reply:
Added parenthesis